docs/poolRegistration.md

```diff
@@ -158,7 +221,9 @@ sudo cardano-hw-cli shelley transaction witness \
 
 ## Create signed transaction 
 
- ```
+Use witnesses from previous step to sign the pool registration transaction
```

Comment:
"to assemble the signed pool registration transaction" would be more accurate as the witnesses are already signatures in their own right